README.md

```diff
@@ -14,12 +10,11 @@ Typical use-case:
 Advantages:
 
  * Improves build time when the Gradle build files themselves are updated, as (most) dependencies are already in the cache
- * Less traffic for artifact repositories (i.e. Maven central, JCenter, your own private Artifactory etc.)
+ * Much less traffic for artifact repositories (i.e. Maven central, JCenter, your own private Artifactory etc.)
 
 Disadvantages:
 
- *  Uses internal Gradle classes to delete unused dependencies. 
-      * The cache job might break on a future version of the Gradle wrapper. If so, it is trivial to (temporarily) revert to the official Gradle orb.
+ *  Uses internal Gradle classes to delete unused dependencies for Gradle versions < 8. For Gradle version >= 8 it uses the built in cache cleanup strategy (with 1 day retention of unused dependencies).
```

Comment:
Is it better to use an earlier version of this orb for Gradle < 8?

Reply:
No, not really. I did consider just supporting gradle 8+ in the latest version, but supporting all versions in the same orb makes for a smoother transistion.